services/dkg/pedersen/mod.go

```diff
@@ -166,9 +166,9 @@ type Actor struct {
 	factory     serde.Factory
 	service     ordering.Service
 	context     serde.Context
-	electionFac serde.Factory
+	formFac serde.Factory
```

Comment:
same comment about formatting. Your IDE should automatically do `gofmt` for you, which is the standard code formater for go.